components/Buttons/src/MDCButton.m

```diff
@@ -301,6 +301,7 @@ - (void)commonMDCButtonInit {
 
   // Set up ink layer.
   _inkView = [[MDCInkView alloc] initWithFrame:self.bounds];
+  _inkView.usesLegacyInkRipple = NO;
```

Comment:
Currently the new ink style halves the opacity of button highlighted state overlays. Is this intended?

Reply:
Discussed this with design. In the tap and hold state, ink now reaches the fully opacity level of the specified inkColor. Ink will purely be a decorative element and will not affect the state of the button or serve as a visual means to communicate a state transition.